ui/src/components/PageNotFound/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  EmptyState,
+  EmptyStateVariant,
+  Title,
+  Button,
+  EmptyStateIcon
+} from '@patternfly/react-core';
+import { IconSize } from '@patternfly/react-icons';
+import { useHistory } from 'react-router-dom';
+import { Icons } from '../../common/icons';
+import Icon from '../Icon';
+import './PageNotFound.css';
+
+export const NotFound: React.FC = () => {
```

Comment:
how about renaming `NotFound` to `PageNotFound` in order to make it consistent with the component package ?